src/login/components/LoginApp/ResetPassword/ExtraSecurity.js

```diff
@@ -57,26 +60,32 @@ function ExtraSecurity(props){
     }else history.push(`/reset-password/`)
   },[extraSecurity]);
 
+  const ShowSecurityKey = (e) => {
```

Comment:
Johan shared a link about making security keys work that might be relevant here. I'll find it and tag you on slack.